src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -100,7 +100,7 @@ public X11XamlRootHost(X11WindowWrapper wrapper, Window winUIWindow, XamlRoot xa
 		_configureCallback = configureCallback;
 
 		_renderTimer = new DispatcherTimer();
-		_renderTimer.Interval = new TimeSpan(1000 / 16);
+		_renderTimer.Interval = TimeSpan.FromMilliseconds(16.66); // 60 hz
```

Comment:
We probably should be setting this to be configurable.

Reply:
Like a `FeatureConfiguration` option? I would rather not expand `FeatureConfiguration` even more, it's already quite bloated. I would guess that 60 fps is smooth enough. Is there a specific scenario where beefier machines could benefit from the extra frames?

Reply:
It's mostly for less performant machines than anything else at this point. I'd expose it though the platform builder, though.